src/devices/Ssd1331/README.md

```diff
@@ -0,0 +1,21 @@
+# Solomon Systech SSD1331 OLED display family
+
+The SSD1331 is a single-chip CMOS OLED/PLED driver with controller for organic/polymer light emitting diode dot-matrix graphic display system. It consists of 96 segments and 64 commons. This IC is designed for Common Cathode type OLED panel.
+
+## Documentation
+
+- HiLetgo 0.95" Inch 7 Pin Colorful 65K SPI OLED Display Module SSD1331 96 * 64 Resolution(https://www.amazon.com/HiLetgo-Colorful-Display-SSD1331-Resolution/dp/B0711RKXB5)
+- SSD1306 [datasheet](https://cdn-shop.adafruit.com/datasheets/SSD1331_1.2.pdf)
+
+## Board
+
+![Connection schematics](Ssd1331.Sample.png)
+
```

Comment:
Please add a short example on how to use it.

Reply:
added